packages/backend/src/constants.ts

```diff
@@ -19,6 +19,7 @@ const Cookies = {
   ClientUat: '__client_uat',
   Handshake: '__clerk_handshake',
   DevBrowser: '__clerk_db_jwt',
+  InfiniteRedirectionLoopCookie: '__clerk_redirection_loop',
```

Comment:
nit: maybe a better name

```suggestion
  RedirectCount: '__clerk_redirect_count',
```

Reply:
yup that looks better 👍🏻